spec/core/data_structures.md

```diff
@@ -416,28 +416,39 @@ func SumTruncated(bz []byte) []byte {
-| validator | [ValidatorParams](#validatorparams) | Parameters limiting the types of public keys validators can use.             | 3            |
-| version   | [BlockParams](#blockparams)         | The ABCI application version.                                                | 4            |
+| Name      | Type                                | Description                                                                               | Field Number |
+|-----------|-------------------------------------|-------------------------------------------------------------------------------------------|--------------|
+| block     | [BlockParams](#blockparams)         | Parameters limiting the size of a block and time between consecutive blocks.              | 1            |
+| evidence  | [EvidenceParams](#evidenceparams)   | Parameters limiting the validity of evidence of byzantine behaviour.                      | 2            |
+| validator | [ValidatorParams](#validatorparams) | Parameters limiting the types of public keys validators can use.                          | 3            |
+| version   | [BlockParams](#blockparams)         | The ABCI application version.                                                             | 4            |
+| synchrony | [SynchronyParams](#synchronyparams) | SynchronyParams influence the validity of block timestamps.                               | 5            |
+| timeout   | [TimeoutParams](#timeoutparams)     | TimeoutParams configure the timings of the steps of the Tendermint consensus algorithm.   | 6            |
+| ABCI      | [ABCIParams](#abciparams)           | ABCIParams configure ABCI functionality specific to the Application Blockchain Interface. | 7            |
+
+
+### HashedParams
+Hashed Params is a subset of Consensus Params, it is amino encoded and hashed into the Header.ConsensusHash.
+
+| Name            | Type  | Description                                                                                                                                                                                                 | Field Number |
+|-----------------|-------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|
+| block_max_bytes | int64 | Max size of a block, in bytes.                                                                                                                                                                              | 1            |
+| block_max_gas   | int64 | Max sum of `GasWanted` in a proposed block. NOTE: blocks that violate this may be committed if there are Byzantine proposers. It's the application's responsibility to handle this when processing a block! | 2            |
```

Comment:
```suggestion
| block_max_gas   | int64 | Maximum sum of `GasWanted` in a proposed block. NOTE: blocks that violate this may be committed if there are Byzantine proposers. It's the application's responsibility to handle this when processing a block! | 2            |
```

Reply:
I would consider taking the "NOTE" out of the table, for better readability